server/src/services/blogs/service.js

```diff
@@ -0,0 +1,83 @@
+import mongoose from 'mongoose';
+import Blogs from '../../models/blogs';
+
+const saveBlog = (req, res, next) => {
+  const { title } = req.body;
+  const { content } = req.body;
+  const userId = req.user._id;
+
+  const blog = {
+    title,
+    content,
+    userId,
+  };
+
+  Blogs.create(blog, (err, blogFromDB) => {
+    if (err) {
+      res.status(500).json({ msg: 'Something went wrong' });
+    }
+    req.blog = blogFromDB;
+    next();
+  });
+};
+
+const fetchBlog = (req, res, next) => {
+  const blogID = req.params.id;
```

Comment:
fetchBlog name suggests doing some HTTP request but I guess right now this is getting blog from database. Better name can be getBlog.